elliott_wave_analyzer.py

```
@@ -697,7 +726,64 @@ def _generate_trading_signals(self, daily_analysis: Dict, weekly_analysis: Dict)
+            else:
+                momentum_type = 'NEUTRAL'
+            
+            return {
+                'momentum': momentum_type,
+                'strength': momentum_strength,
+                'confidence': regime_confidence,
+                'market_regime': market_regime,
+                'filter_active': momentum_type == 'STRONG_BULL' and momentum_strength > self.momentum_threshold
+            }
+            
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)